var/spack/repos/builtin/packages/spatial-index/package.py

```diff
@@ -16,7 +15,9 @@ class SpatialIndex(PythonPackage):
     depends_on("cmake")
     depends_on("boost")
 
-    version('0.2.0', tag='0.2.0', submodules=True)
+    version('0.2.1', tag='0.2.1', submodules=True)
     version('0.1.0', tag='0.1.0', submodules=True)
 
-
+    @run_after('install')
```

Comment:
Hmm, I guess it's either that or we make it a CMakePackage and jump through additional hoops to get the Python setup right :/

Reply:
yeah...this is "non-standard" (using post-install hook to "complete" part of installation)

Reply:
We could override install(), call `super().install()` and the `install_tree`... But IMO it's complicating...

Reply:
It's fine as it is, a nice solution :)

I was just lamenting the extra complexity necessary with either Python and CMake packages to get hybrids dealt with.